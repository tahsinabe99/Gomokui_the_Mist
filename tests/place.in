place A1
place 1A
place A1
A1 place
place B1
who
Never gonna give you up
resign
  