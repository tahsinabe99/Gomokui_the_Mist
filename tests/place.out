Invalid coordinate
Occupied coordinate
Invalid!
B
Invalid!
White wins!
A1B1
Thank you for playing!
